lessons/Algebra2/exploring-exponential-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,200 @@
+= Exploring Exponential Models
+
+@description{Students investigate exponential relationships in data about Covid spread, using an inquiry-based model of sense-making, hypothesizing, modeling and testing. They are introduced to table transformations and inverse functions, which are used to fit exponential models onto nonlinear data.}
+
+@lesson-prereqs{exploring-quadratic-models}
+
+@keywords{exponential, quadratic, square, coefficient}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Read and interpret real-world data, presented in a scatter plot
+* Model exponential relationships using functions
+
+| Student-facing Lesson Goals
+|
+
+* Let's use Pyret to model exponential relationships in data
+* Let's use Pyret to filter a dataset into subsets
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Preparation
+|
+* All students should pull up their @online-exercise{program-list} and, open the @starter-file{alg2-covid}.
+
+| Key Points For The Facilitator
+|
+* TBD
+|===
+
+== Filtering {amp} Simpson's Paradox @duration{20 minutes}
+
+=== Overview
+Students explore the @starter-file{alg2-covid}, and discover that some scatter plots show __multiple__ trends instead of just one! This motivates a brief discussion of Simpson's Paradox, and teaches them to filter their dataset into grouped samples.
+
+=== Launch
+
+[.lesson-instruction]
+- Open the @starter-file{alg2-covid}
```

Comment:
I think it's worth telling a little about the origin story of this dataset. Might as well tell people it only focuses on New England before they're disappointed not to be able to learn about their geography. Also, what's special about June 20? Seems pretty random. Even stating that the data in this dataset spans June 20, 2020 to ____, would set it up to make more sense for June 21 to be day 1.

Reply:
I've added a more in-depth explanation in the "Going Deeper" pedagogy box, which I've renamed and moved to the top of the lesson (where teachers and students are likely to ask that question!).

I also added a discussion of the README stuff to our next one-on-one